tool/cli.js

```diff
@@ -891,6 +893,54 @@ if (Mozilla && !Mozilla.dntEnabled()) {
     })
   )
 
+  .command(
+    "frontmatter-inventory",
+    "Create frontmatter content inventory as JSON"
+  )
+  .action(
+    tryOrExit(async ({ options }) => {
+      if (options.verbose) {
+        console.log(chalk.green("Checking for precense of CONTENT_ROOT"));
+      }
+
+      if (!CONTENT_ROOT) {
+        throw new Error(
+          "CONTENT_ROOT not set. Please see the docs at tool/README.md"
+        );
+      }
+
+      if (options.verbose) {
+        console.log(chalk.green("CONTENT_ROOT set, generating inventory"));
+      }
+
+      const inventory = {};
+      const allPaths = klawSync(CONTENT_ROOT, {
+        nodir: true,
+        filter: (entry) => path.extname(entry.path) === ".md",
+        traverseAll: true,
+      });
+
+      allPaths.forEach((entry) => {
+        const fileContents = fs.readFileSync(entry.path, "utf8");
+        const parsed = frontmatter(fileContents);
+        inventory[entry.path.substring(entry.path.indexOf("/files"))] = {
+          frontmatter: parsed.data,
+        };
+      });
+
+      if (options.verbose) {
+        console.log(chalk.green("Writing ./frontmatter-invetory.json"));
+      }
+      fs.writeFileSync(
```

Comment:
What do you think of writing to standard output instead? Then I could pipe this to other tools (especially `jq`) directly. To make this work, I think all the existing `console.log` calls would need to become `console.error` to write to standard error, as to not wreck the JSON output.